staticDHCPd/extensions/official/httpdb.py

```diff
@@ -263,12 +266,34 @@ def lookupMAC(self, packet_or_mac, packet_type=None, mac=None, ip=None,
 
 class HTTPCachingDatabase(CachingDatabase, _HTTPLogic):
     def __init__(self):
+        from staticdhcpdlib import config
```

Comment:
Correct. It needed to check the concurrency limit to initialize the CachingDatabase.